lib/browser/browser.ts

```diff
@@ -222,16 +275,17 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
   // Revokes the access token for the application session
   async revokeAccessToken(accessToken?: AccessToken) {
     if (!accessToken) {
-      accessToken = await this.tokenManager.get(ACCESS_TOKEN_STORAGE_KEY) as AccessToken;
-      this.tokenManager.remove(ACCESS_TOKEN_STORAGE_KEY);
+      accessToken = (await this.tokenManager._getTokens()).accessToken as AccessToken;
+      const accessTokenKey = this.tokenManager._getStorageKeyByType('accessToken');
```

Comment:
should use a constant instead of a string for "accessToken" ?

Reply:
added type in https://github.com/okta/okta-auth-js/pull/453/files#diff-8163593358d0d73dd6138e68ffadcd32R38, I felt it can be detected by typescript anyway, then just left the string here.